cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailureDetector.java

```diff
@@ -44,8 +44,12 @@
 public class BrokerFailureDetector {
   private static final Logger LOG = LoggerFactory.getLogger(BrokerFailureDetector.class);
   public static final String FAILED_BROKERS_OBJECT_CONFIG = "failed.brokers.object";
+  public static final String BROKER_FAILURES_FIXABLE_CONFIG = "broker.failures.fixable.object";
```

Comment:
The role of this config is not clear from the name. Should we add a comment to clarify?